jax/_src/numpy/array.py

```diff
@@ -0,0 +1,376 @@
+# Copyright 2025 The JAX Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib
+from typing import Any
+
+import numpy as np
+
+from jax._src import api
+from jax._src import config
+from jax._src import core
+from jax._src import dtypes
+from jax._src import tree_util
+from jax._src import xla_bridge
+from jax._src.basearray import Array as ndarray  # noqa: F401
```

Comment:
OOC: What's the story here?

Reply:
Ah, I guess I probably shouldn't have done this here. But my thinking was "what other objects make sense in `jax/_src/numpy/array.py` and `ndarray` seemed like a good choice. But I think I'll revert this since it complicates the change